### ISSUE TITLE
Exiv2 Developer's Meeting [proposal]

### ISSUE THREAD

**Issue by clanmills** [Exiv2/exiv2#225]
Weekend of Saturday May 5

Venue:
Robin's Home in England (I'll provide the address by private email)

Objectives:
The primary objective of the meeting is to get to know each other better.  The secondary objective is to agree on the contents and schedule for v0.27.

Attendees:
Robin and Alison will provide accommodation and meals for participants and partners.  If anybody wants to bring their children, I'll investigate AirBnB and the local hotel.  Robin and Alison have 4 bedrooms and can stay with neighbours if we have 4 participants.  If we have more than 4 participants, I'll investigate AirBnB and the local hotel.

Location:
We live in Camberley, Surrey.  It's about 30km from Heathrow Airport.  We will collect/drop-off participants at the airport.  Gatwick is one hour by train from Camberley.  Don't consider Luton, Stanstead or London City Airport as they are very inconvenient for Camberley. 

Program:
Try to arrive by Friday so we have a full day together on Saturday.
Monday is a public holiday in England.
There is a 10K event in Camberley on Sunday.   If everybody's staying until Monday, we could run on Sunday morning.

Participants are welcome to stay with us before/after the meeting if want to have a vacation in England that includes the meeting.

Participant's partners will be entertained by Alison while the engineers are discussing technology.  The Royal Wedding on May 19 will be in Windsor which is 20km from Camberley.  A walk in Windsor to see the preparations is highly likely.  Camberley is 35 minutes from London by train from Brookwood Station.  We'll give you a lift to/from Brookwood if partners want to visit London.

I hope we have a successful meeting and enjoyable time together.  Exiv2 has been a considerable burden for years and I believe the project is now ready to move into the future as a team effort.  I hope this meeting will lay the foundations for a successful future.

**Comment by clanmills** [comment]
Here are my notes in preparation for the meeting.  They will be revised almost daily in April.  http://dev.exiv2.org/news/3

**Comment by clanmills** [comment]
The meeting was very productive and enjoyable.  We didn’t work all the time.  We had dinner on Friday night, ran 10K on Saturday morning and had a BBQ on the Patio on Saturday Evening.

We had a discussion about the (GPLv2 and commercial) license after dinner on Friday.  We discussed v0.27 on Saturday.  We didn’t run on Sunday (the young guys couldn’t face it) and discussed v0.28 projects.  I will publish my notes here shortly.
 
Our priority for v0.27 is security/fuzzing and we’ll release v0.27.1 and other dots if necessary in 2019.  I’d like Luis to focus on BigTiff (64 bit tiff) for v0.28.  I don’t recall us agreeing on a “top priority” for Dan for v0.28.  I’m going to do “User Support” and then retire as I’ll be 69 at the end of 2019.

We discussed lens recognition and we don’t think we have the resources to re-engineer that code in the next 12 months.  With the project moving steadily, we hope to recruit another engineer to be “lens guy” and undertake that project.

Alison and I really enjoyed meeting Luis and Dan and his girl-friend Elizabeth.  Will we have more meetings in future?  I don’t know.  Our home is a good venue, however I don’t want to burden them with the expense of travelling.
